### PR TITLE
[DEV-1279] seed backup

### PR DESCRIPTION
**Pull request by Krysh90**

### CODE REVIEW

File: components/Checkbox.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    onChanged(isClicked);
+  }, [isClicked]);
+
+  return (
+    <TouchableOpacity style={styles.container} onPress={() => setIsClicked(!isClicked)}>
+      <Icon name={isClicked ? 'checkbox-outline' : 'checkbox-blank-outline'} type="material-community" size={20} color={'#F5516C'} />
+      <Text style={styles.text}>{text}</Text>
+    </TouchableOpacity>
+  );
+}
```
Comment (davidleomay) [inline components/Checkbox.tsx:35]: Check with Oli if we can use the built-in element.

**Review by davidleomay — APPROVED** [review]

**Review by davidleomay — COMMENTED** [review]
Talked to Oli, let's replace the custom Checkbox with the default RN switch.

**Comment by Krysh90** [comment]
> Talked to Oli, let's replace the custom Checkbox with the default RN switch.

Okay, don't really understand the change, but if we want to change it - I will do it

**Comment by Krysh90** [comment]
> > Talked to Oli, let's replace the custom Checkbox with the default RN switch.
> 
> Okay, don't really understand the change, but if we want to change it - I will do it

I think now I got it, what you want to achieve. There is a default CheckBox element from react-native-elements. Haven't seen it. Will change it accordingly

**Review by davidleomay — APPROVED** [review]

**Pull request merged**